### PR TITLE
Added methods

### PR DESCRIPTION
**Pull request by sebalp100**
# Added methods

For this project, we made a code that allows the user to use different enumerables methods.
We divided the project as follows:

- An enumerable file that contains the enumerables any, all, and filter methods
- A my_list file that contains the list class code and uses each to iterate over each list element
- Added ruby linters and Rubocop

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require './my_enumerable'
```
Comment (Felix45) [inline my_list.rb]: - [ Required ] Good job implementing a working `enumerable`, I just have one observation that can improve your code. The `require` statement is usually used to import `external files` like `ruby gems`, so as part of the ruby convention if you want to import files within your directory you should use `require_relative` statement. In light of this please consider refactoring this line as shown below.

```ruby
  require_relative 'my_enumerable'
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @sebalp100 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 
- [ ] Enumerable works as expected :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (sebalp100) [reply my_list.rb]: @Felix45 Thank you for your feedback. I will implement this change 👍

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @sebalp100 & @osbentech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Enumerable works as expected :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 
- [ ] Descriptive pull request summary :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**